src/state/navigation/extensionsAtom.ts

```diff
@@ -177,7 +180,10 @@ const getExtensionWizards = async (
             convertYamlToObject,
           ) as ExtResource,
         };
-
+        if (extCustomComponentsEnabled) {
+          extResourceWithMetadata.data.customHtml =
+            extResourceWithMetadata.data.customHtml || '';
+        }
```

Comment:
You are copy-pasting the same value with only the `|| ''` change, is it really necessary? Can't we handle this value somehow differently?